integration/test.bats

```diff
@@ -29,7 +29,7 @@ wait_for() {
 
 @test "it prints the version" {
     run "${BATS_TEST_DIRNAME}/../supercronic" -version
-    [[ "$output" =~ ^v1337$ ]]
+    [[ "$output" != "<unset> ]]
```

Comment:
Missing a closing quote. Did you try `make integration` before submitting?

Reply:
Ah crap, I did run `make integration` but only on my local repo and I synced up my changes with this fork by hand... Whoops